charts/ccd-definition-store-api/values.demo.template.yaml

```diff
@@ -0,0 +1,17 @@
+java:
+  keyVaults:
+    ccd:
+      secrets:
+        - definition-store-api-POSTGRES-PASS-V11
+        - ccd-definition-s2s-secret
+        - ccd-ELASTIC-SEARCH-URL
+        - ccd-ELASTIC-SEARCH-PASSWORD
+        - storage-account-primary-connection-string
+        - AppInsightsInstrumentationKey
+  environment:
+    FLYWAY_NOOP_STRATEGY: true
```

Comment:
`FLYWAY_NOOP_STRATEGY: true`
do we need this? it's already in `values.yml`

Reply:
Demo deployment is by flux. You need to add to flux (demo.yaml)